common-content/en/module/onboarding/help/index.md

```diff
@@ -0,0 +1,51 @@
++++
+title="🧭 Help"
+description="Make sure you have everything you need to come to class and do your work"
+hide_from_overview="true"
+time=30
+objectives = [
+  "Find the Signposts website",
+  "List the financial learner support available to you"
+]
+[build]
+  render = 'never'
+  list = 'local'
+  publishResources = false
++++
+
+🧭 _Everyone_ at {{<our-name>}} has barriers to education, and we can [help with some things](https://signposts.codeyourfuture.io/). If you cannot come to an in-person class because you can't afford:
```

Comment:
Can we clarify here that this help is available once you have enrolled as a trainee, please?